### DISCUSSION TITLE
Leave Application

### DISCUSSION THREAD

**Discussion by Mari-tech2000** [horilla-opensource/horilla#254] [General]
On applying leave from a employee login i'm getting an error like this
![image](https://github.com/user-attachments/assets/fd875235-87f6-48ef-91b1-b2689ba00981)

**Comment by horilla-opensource** [comment]
> On applying leave from a employee login i'm getting an error like this ![image](https://private-user-images.githubusercontent.com/59371105/349239286-fd875235-87f6-48ef-91b1-b2689ba00981.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExOTE5MDksIm5iZiI6MTcyMTE5MTYwOSwicGF0aCI6Ii81OTM3MTEwNS8zNDkyMzkyODYtZmQ4NzUyMzUtODdmNi00OGVmLTkxYjEtYjI2ODliYTAwOTgxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE3VDA0NDY0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAxNDJhMDY5OWZkZDg5YWQ1YTQ4MTQ4YmU5YTA3YzdlMjU2OGRmMjBiOTdkNWVjNjQyMzQ5ZWYxZDAyN2U5YzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wn9anzEnEsFtGBnyN4M1LRjX6ld9mNe88_QbH0USE-o)

Hi @Mari-tech2000 ,
Is the form not loading for creating leave request ?

With Regards,
**Team Horilla**
